### PR TITLE
add a first login function and test 3.1.1

### PR DESCRIPTION
**Pull request by HaGuesto**

### CODE REVIEW

File: cypress/support/session.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Standard Login method for all tests
+Cypress.Commands.add("LoginAjax", (userMail, userPassword, autologin) => {
+    cy.visit('/'); //not sure if needed. Somehow it does not work without it.
```
Comment (HaGuesto) [inline cypress/support/session.js]: @jamescrowley This is the wicked bit. I would like to get rid off this line, but it does not work otherwise.

Comment (jamescrowley) [reply cypress/support/session.js]: Pushed two commits that resolve this - need to use cy.request :) rather than making the AJAX request via jQuery...

**Comment by jamescrowley** [comment]
@HaGuesto did you mean to closed this? I'd pushed the commits which fixed this issue on the branch, but they haven't been merged. And I don't think your tests have either?

**Comment by jamescrowley** [comment]
Ah, I'll just close and add my commits again in a separate PR - but let me know if you need your tests in too

**Pull request closed without merging**